tools/src/commands/LegacyUpdateVendoredModule.ts

```diff
@@ -0,0 +1,886 @@
+
+      const javaFiles = await findAndroidFilesAsync(sourceDir);
+
+      await copyFilesAsync(javaFiles, sourceDir, targetDir);
+
+      const files = await findAndroidFilesAsync(targetDir);
+
+      for (const file of files) {
+        await renamePackageAndroidAsync(file, step.sourceAndroidPackage, step.targetAndroidPackage);
+      }
+    }
+  }
+  const { name, version } = await JsonFile.readAsync<{
+    name: string;
+    version: string;
+  }>(path.join(tmpDir, moduleConfig.packageJsonPath ?? '', 'package.json'));
+  const semverPrefix =
+    (options.semverPrefix != null ? options.semverPrefix : moduleConfig.semverPrefix) || '';
+  const versionRange = `${semverPrefix}${version}`;
+
+  await updateBundledNativeModulesAsync(async (bundledNativeModules) => {
+    if (moduleConfig.installableInManagedApps) {
+      bundledNativeModules[name] = versionRange;
+      console.log(
+        `Updated ${chalk.green(name)} in ${chalk.magenta(
+          'bundledNativeModules.json'
+        )} to version range ${chalk.cyan(versionRange)}`
+      );
+    } else if (bundledNativeModules[name]) {
+      delete bundledNativeModules[name];
+      console.log(
+        `Removed non-installable package ${chalk.green(name)} from ${chalk.magenta(
+          'bundledNativeModules.json'
+        )}`
+      );
+    }
+    return bundledNativeModules;
+  });
+
+  console.log(`\nUpdating ${chalk.green(name)} in workspace projects...`);
+
+  await Workspace.updateDependencyAsync(name, versionRange);
+
+  console.log(`\nRunning \`${chalk.cyan(`yarn`)}\`...`);
+
+  // We updated dependencies so we need to run yarn.
+  await Workspace.installAsync();
+
+  console.log(
+    `\nFinished updating ${chalk.green(
+      moduleName
+    )}, make sure to update files in the Xcode project (if you updated iOS, see logs above) and test that it still works. 🙂`
+  );
+}
+
+export default (program: Command) => {
+  program
+    .command('update-vendored-module')
+    .alias('update-module', 'uvm')
+    .description('Updates 3rd party modules.')
+    .option('-l, --list', 'Shows a list of available 3rd party modules.', false)
+    .option('-o, --list-outdated', 'Shows a list of outdated 3rd party modules.', false)
+    .option('-m, --module <string>', 'Name of the module to update.')
+    .option(
+      '-p, --platform <string>',
+      'A platform on which the vendored module will be updated.',
+      'all'
+    )
+    .option(
+      '-c, --commit <string>',
+      'Git reference on which to checkout when copying 3rd party module.',
+      'master'
+    )
+    .option(
+      '-s, --semver-prefix <string>',
+      'Setting this flag forces to use given semver prefix. Some modules may specify them by the config, but in case we want to update to alpha/beta versions we should use an empty prefix to be more strict.',
+      null
+    )
+    .option('--no-pbxproj', 'Whether to skip updating project.pbxproj file.', false)
+    .asyncAction(action);
+};
```

Comment:
I'm not checking this file, as it's name suggests it's leftover from the ongoing process of refactoring this script, am I right? 🤔

Reply:
This is in the old, legacy version of the command where the vendored modules were added directly to the project and not as a pod and because of that any new files have to be added to pbxproj file. The new vendoring command won't need to do this tho.